jest.config.js

```diff
@@ -3,10 +3,13 @@ module.exports = {
   testEnvironment: 'node',
   roots: ['<rootDir>'],
   moduleDirectories: ['node_modules', 'server'],
-  globals: {
-    'ts-jest': {
-      tsconfig: './tsconfig.test.json',
-    },
+  transform: {
+    '.(ts|tsx)': [
```

Comment:
Btw, I don't know the difference in regex, but here is the one in ts-jest docs https://kulshekhar.github.io/ts-jest/docs/getting-started/options

```suggestion
    '^.+\\.tsx?$': [
```